astropy/nddata/nddata.py

```diff
@@ -140,32 +139,32 @@ def __init__(self, data, uncertainty=None, mask=None, wcs=None,
             # other parameters.
             if (unit is not None and data.unit is not None and
                     unit != data.unit):
-                log.info("Overwriting NDData's current "
-                         "unit with specified unit")
+                log.info("overwriting NDData's current "
```

Comment:
I can't comment on grammatical issues, either is fine by me. Just had this vague memory of someone mentioning this as a pythonic convention, when I looked at @MSeifert04's other PR in ccdproc.